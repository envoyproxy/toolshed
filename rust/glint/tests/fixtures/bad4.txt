	All lines start with tabs
	So no mixed indentation
	But this one has trailing whitespace	
	And this file ends without a newline