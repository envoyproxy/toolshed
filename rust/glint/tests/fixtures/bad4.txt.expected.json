{
  "files": {
    "tests/fixtures/bad4.txt": {
      "trailing_whitespace": [
        3
      ],
      "no_final_newline": true
    }
  },
  "summary": {
    "total_files": 1,
    "files_with_issues": 1,
    "total_issues": 2
  }
}
